From b0d4ef3c575420792a87d113bde95a0d5220f296 Mon Sep 17 00:00:00 2001
From: Lancelot Owczarczak <contact@lancelot.life>
Date: Sat, 2 May 2020 20:22:55 +0200
Subject: [PATCH] kernel build: add -thor_kernel in EXTRAVERSION

Change the EXTRAVERSION variable in the Makefile to add -thor_kernel

Signed-off-by: Lancelot Owczarczak <contact@lancelot.life>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 679f302a8b8b..e4b3e2807d4d 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 5
 PATCHLEVEL = 7
 SUBLEVEL = 0
-EXTRAVERSION = -rc3
+EXTRAVERSION = -rc3-thor_kernel
 NAME = Kleptomaniac Octopus
 
 # *DOCUMENTATION*
-- 
2.26.2

